File: CodeForces/1-100/4/A/4A.pl
#!/usr/bin/perl

#! Problem 4A: Watermelon
#! https://codeforces.com/contest/4/problem/A
#! Executed with: perl 5, version 36, subversion 0 (v5.36.0) built for x86_64-linux-gnu-thread-multi
#! Linux version: Linux kernel 6.1.0-10-amd64
#! Submitted on: October 22nd, 2025
#! Codeforces language used: Perl 5.20.1

use strict;
use warnings;

my $w = <STDIN>;
chomp($w);

if ($w > 2 && $w % 2 == 0) {
    print "YES\n";
} else {
    print "NO\n";
}

File: CodeForces/1-100/4/A/4A.hs
-- Problem 4A: Watermelon
-- https://codeforces.com/contest/4/problem/A
-- Executed with: The Glorious Glasgow Haskell Compilation System, version 9.6.6
-- Linux version: Linux kernel 6.17.0-6-generic
-- Submitted on: October 28th, 2025
-- Codeforces language used: Haskell GHC 8.10.1

main :: IO ()
main = do
    line <- getLine
    let w = read line :: Int
    if w > 2 && w `mod` 2 == 0
        then putStrLn "YES"
        else putStrLn "NO"
